eventstore/model_test.go

```diff
@@ -372,7 +373,7 @@ func TestInvalidActions(t *testing.T) {
 	})
 	t.Run("Save", func(t *testing.T) {
 		p := &PersonFake{Name: "fake"}
-		model.Create(p)
+		_ = model.Create(p)
```

Comment:
Uhm, could be checked too.

Reply:
Will update

Reply:
I had to fiddled with the test logic a little to get it to pass correctly